stat/int.go

```diff
@@ -0,0 +1,27 @@
+package stat // import "github.com/influxdata/influxdb/stat"
+
+import "sync"
+
+type Int struct {
+	mu    sync.RWMutex
```

Comment:
The reason why I stopped using `sync/atomic` was because it caused alignment issues on i386. When I did that originally though, I was doing this:

```
statMap struct {
    Stat1 int64
    Stat2 int64
}
```

The second cause a panic due to an alignment issue. Will I still get this alignment issue with these structs?